### PR TITLE
Basic Changes

### PR DESCRIPTION
**Pull request by agr-ayush**

### CODE REVIEW

File: HelperViews/ContextMenuActions.cs [hunk 1]
```diff
@@ -59,5 +60,70 @@ public static void SendToBackButton_Clicked(View targetElement, ContextMenu cont
...
+            }
+            contextMenu.Close();
+        }
+
+        private static View CloneView(View originalView)
+        {
+            
+            var newView = ElementCreator.Create(originalView.GetType().Name);
+
+            newView.BindingContext = originalView.BindingContext;
```
Comment (GMPrakhar) [inline HelperViews/ContextMenuActions.cs:102]: Does this mean that they have same inherent context?

**Review by GMPrakhar — APPROVED** [review]

**Comment by GMPrakhar** [comment]
Please resolve conflict so that we can merge !

File: HelperViews/ContextMenuActions.cs (hunk 1 above)
Comment (agr-ayush) [reply HelperViews/ContextMenuActions.cs:102]: Paste is not working perfectly. we still have to make it work.

**Comment by agr-ayush** [comment]
> Please resolve conflict so that we can merge !

I don't see any conflicts. Also, i do not see option to merge.

**Pull request merged**